CHANGES.md

```diff
@@ -125,6 +125,14 @@ after the corresponding tracker issues.
   override this choice.
   <http://issues.fast-downward.org/issue1031>
 
+- For users: delete-relation constraints can now be used in the
+  operator-counting framework.
+  <http://issues.fast-downward.org/issue983>
+  The constraints defined by Imai and Fukunaga (JAIR 2015) encode
+  different relaxations of the delete-relaxation heuristic.
+  For details, see our documentation.
+  <http://www.fast-downward.org/Doc/ConstraintGenerator#Delete_relaxation_constraints>
+
```

Comment:
Shouldn't the change log also mention that a bug was fixed for operator-counting constraints using integer variables? Perhaps I'd make it a separate entry because it is unrelated from a user perspective even though it was handled in the same issue. I see no harm in having two entries that point to the same issue.